src/code/z_room.c

```diff
@@ -265,17 +253,17 @@ void Room_DrawCullable(PlayState* play, Room* room, u32 flags) {
         }
 
         if (flags & ROOM_DRAW_XLU) {
-            for (; head != NULL; head = head->prev) {
+            for (; tail != NULL; tail = tail->prev) {
```

Comment:
so the non equivalency was using `tail` instead of `head`?

Reply:
Yeah, OPA goes forward through the list, XLU goes in reverse. Also if both OPA and XLU flags are set, this now goes through the XLU list as well. Using head would not since head would already be NULL from the OPA walk through.